docs/framework-components/core-components/commands.mdx

```diff
@@ -0,0 +1,239 @@
+        </div>
+    </div>
+);
+
+The Commands Component offers a mechanism for clients to request Sparkplug CMD messages be issued to other clients, given appropriate authorisation. The Sparkplug specification indicates that only the Primary Application is responsible for transmitting CMDs but the precise methodology and timing for this process are not explicitly defined. It is inferred that applications must interface with the Primary Application to request CMDs and as such the Commands component provides the implementation for this specific aspect of the Primary Application's functionality.
+
+
+<OpenSourceExample buttonText={'ACS Commands Component'} repoUrl={'https://github.com/AMRC-FactoryPlus/acs-cmdesc'}></OpenSourceExample>
+
+## Overview
+
+<ComponentOverview></ComponentOverview>
+
+## MQTT Interface
+
+:::caution Ensure topic ACLs are configured correctly
+Command requests via the Sparkplug interface are authenticated on the basis of the Node address they are sent from. The address is looked up in the Configuration Store, and the UUID of the entry is treated as the principal to authorise. This relies utterly on the MQTT ACLs being set up appropriately (e.g. if Node A has permission to publish to the `DDATA`/`NDATA` topics of Device B then it could imitate a command request).
+:::
+
+### Specification
+- The Commands component **MUST** listen to all DATA topics
+- Any DATA packet containing a metric named `Execute_Remote_Command` (at the top level) **MUST** be treated as a command escalation request
+- Any Node that wishes to publish the `Execute_Remote_Command` metric **MUST** advertise it in its birth certificate
+- DATA packets **MUST NOT** use aliases
+- Requests **MUST** be sent entirely in a single Sparkplug packet
+
+### Requests
+
+The metric **MUST** be of `Template` type. The Commands component **SHOULD NOT** check the `templateRef` or `version` fields. The template **MUST** have the following fields:
+
+| Name                     | Type     | Value                                              |
+|--------------------------|----------|----------------------------------------------------|
+| `Receivers_Group_ID`     | String   | Group ID to send a command to                      |
+| `Receivers_Edge_Node_ID` | String   | Node ID to send a command to                       |
+| `Receivers_Device_ID`    | String   | Device ID to send a command to, or an empty string |
+| `Tag_Path`               | String   | Metric name to send in the command                 |
+| `Tag_Value`              | String   | Stringified metric value to send                   |
+| `Command_Timestamp`      | DateTime | Timestamp the request was sent                     |
+
+Publishing the `Command_Timestamp` metric should be considered the point at which the request is actually submitted; i.e. the timestamp metric should be sent last.
```

Comment:
```suggestion

```

The timestamp component was there only because Ignition would send packets out one at a time, or handle them incoming one at a time. In our case we can make this field redundant.